lib/FreeRTOS-Plus-TCP/source/FreeRTOS_DNS.c

```diff
@@ -1346,11 +1346,6 @@ TickType_t xTimeoutTime = pdMS_TO_TICKS( 200 );
 		/* For each entry in the DNS cache table. */
 		for( x = 0; x < ipconfigDNS_CACHE_ENTRIES; x++ )
 		{
-			if( xDNSCache[ x ].pcName[ 0 ] == 0 )
-			{
-				break;
```

Comment:
Let's not proceed into the loop if it's an invalid entry, though. Could we replace the break with a continue?

Reply:
Done. The only if statement in the loop would have achieved the same effect, but continue is cleaner.